themes/bootstrap3/templates/search/newitem.phtml

```diff
@@ -21,7 +24,7 @@
     <div class="btn-group" data-toggle="buttons">
       <?php foreach ($this->ranges as $key => $range): ?>
         <label class="btn btn-primary<?php if ($key == 0): ?> active<?php endif ?>">
-          <input type="radio" name="range" id="newitem_range_<?=$this->escapeHtmlAttr($key)?>" value="<?=$this->escapeHtmlAttr($range)?>"<?=($key == 0) ? ' checked="checked"' : ''?>>
+          <input type="radio" name="range" id="newitem_range_<?=$this->escapeHtmlAttr($key)?>" value="<?=$this->escapeHtmlAttr($range)?>"<?=($key == 0) ? ' checked="checked" data-checked-by-default="true"' : ''?>>
```

Comment:
What is "data-checked-by-default" for?

Reply:
When you click the "reset" button on the form, it will restore checkboxes to a checked state if they have this data attribute.

Reply:
See: https://github.com/vufind-org/vufind/blob/dev/themes/bootstrap3/js/advanced_search.js#L169-L173

Reply:
Makes sense.